.ci/test-install-docs.sh

```diff
@@ -84,11 +154,81 @@ test_distro_install_guide()
 	info "Install using ${test_name}"
 
 	create_kata_container "${test_name}"
+
+	# Clean up
+	$mgr remove-packages
+}
+
+# Apart from the distro-specific install guides, users can choose to install
+# using one of the following methods:
+#
+# - kata-manager ("Automatic" method).
+# - kata-doc-to-script ("Scripted" method).
+#
+# Testing these is awkward because we need to "execute" the documents
+# describing those install methods, but since those install methods should
+# themselves entirely document/handle an installation method, we need to
+# convert each install document to a script, then delete all the kata code
+# repositories. This ensures that when each install method script is run, it
+# does not rely on any local files (it should download anything it needs). But
+# since we're deleting the repos, we need to copy this script to a temporary
+# location, along with the install scripts this function generates, and then
+# re-exec this script with an option to ask it to run the scripts the previous
+# instance of this script just generated.
+test_alternative_install_methods()
+{
+	local -a files
+	files+=("installing-with-kata-manager.md")
+	files+=("installing-with-kata-doc-to-script.md")
+
+	local tmp_dir
+
+	tmp_dir=$(mktemp -d)
```

Comment:
Do we ever finally delete the `tmp_dir`? Maybe a `TRAP` hook like we use elsewhere might do it?

Reply:
Yes, I'll fix that for when running with `-t`...

Reply:
Fixed.

Reply:
If it is fixed can you remove the FIXME?